openapi/pacta.yaml

```diff
@@ -222,15 +346,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/User'
-
-        '403':
-          description: caller does not have access or user does not exist 
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/Error'
-
+                $ref: '#/components/schemas/EmptySuccess'
```

Comment:
Ah, now I understand the `emptySuccess`. Pretty sure you can just leave this blank (e.g. no `content`) for actual empty success.